setup.py

```diff
@@ -47,6 +48,8 @@
         'test': [
           'plone.app.testing',
           'plone.app.dexterity',
+          'Products.ATContentTypes',
+          'unittest2',
```

Comment:
And to be clear @hvelarde, I agree that we can drop that support, but we need to make it explicit